services/auth-basic/README.md

```diff
@@ -0,0 +1,22 @@
+# Auth-Basic Service
+
+The `auth-basic` service is responsible for validating authentication of incoming requests. To do so, it will forward the call to an `auth manager`, see the `Auth Managers` section. Almost every request to ocis will involve the `auth-basic` service, as access tokens need to be verified.
```

Comment:
```suggestion
The `auth-basic` service is responsible for validating authentication of incoming requests. To do so, it will forward the call to an `auth manager`, see the `Auth Managers` section. Only HTTP basic auth requests to ocis will involve the `auth-basic` service.
```